src/test/java/com/egov/core-services/tests/userPasswordUpdate.feature

```diff
@@ -0,0 +1,65 @@
+Feature: Update user
```

Comment:
I think better to have Feature file name combination of both Service Name & Endpoint. Like eGovUser-userPasswordUpdate. Just a thought!!

Reply:
If you feel it's good approach then please update in rest other files as well.